text/rocq-roadmap.md

```diff
@@ -49,6 +48,7 @@ This roadmap outlines our vision for Rocq's future, focusing on four key pillars
 
 ### 4. Enhanced Usability
 
+* **Performance**: address performance limitations that impact projects that are widely used in industry and academia.
```

Comment:
For consistency with the rest of the items, there should be target audiences. Given the content of the line, this should be:
```suggestion
* **Performance**: address performance limitations that impact projects that are widely used in industry and academia (C, D).
```